src/Parallel/spanningtrees/boruvka-distributed.jl

```diff
@@ -0,0 +1,129 @@
+#=
+boruvka_multithread_lightgraph:
+- Julia version: 1.3.1
+- Author: hanstananda
+- Date: 2020-03-15
+=#
+@everywhere using LightGraphs, SimpleWeightedGraphs
+using CPUTime
+using DelimitedFiles
+@everywhere using Base
+@everywhere using DataStructures
+using SharedArrays
+
+"""
+    boruvka_mst_distributed(g)
+
+Return a tuple `(mst, weights)` where `mst` is a vector of edges representing the
+optimum (minimum, by default) spanning tree of a connected, undirected graph
+`g` , and `weights` is the sum of all the edges in the solution by using
+[Boruvka's algorithm](https://en.wikipedia.org/wiki/Bor%C5%AFvka%27s_algorithm).
+The algorithm requires that all edges have different weights to correctly generate a minimun/maximum spanning tree
+### Optional parameter(s):
+`maxItr`: Used to limit maximum number of iterations the algorithm will be performed. The default is log2(numVertex).
```

Comment:
camelCase is not generally used in Julia (except for structs). Prefer snake_case.

Reply:
Ah sorry my bad, I'm getting mixed up with the other language naming conventions. I'll change them.